SLO roles privileges changed (#4300)
elastic/observability-docs: docs/en/observability/slo-privileges.asciidoc

IMPORTANT: To create and manage SLOs, you need an {subscriptions}[appropriate license].

You can enable access to SLOs in two different ways:

* Creating the following roles, depending on the type of access needed:
** <<slo-all-access,*SLO Editor*>> — Create, edit, and manage SLOs and their historical summaries.

> ** <<slo-read-access,*SLO Viewer*>> — See SLOs and their historical summaries.

Reviewer: ```suggestion
** <<slo-read-access,*SLO Viewer*>> — Check SLOs and their historical summaries.
```
Replace visual clues with more accessible terms.
For more info, refer to [How to create accessible web content](https://docs.google.com/document/d/1IGuMf16-cwxx8BX1Z9PWsFWsTPxicwTkX_r7UOhjd4E/edit#heading=h.iykpo698d18b).

* Using the `editor` {ref}/built-in-roles.html[built-in role]. This role grants full access to all features in {kib} (including the {observability} solution)
and read-only access to data indices. Users assigned to this role can create, edit, and manage SLOs.
+
[NOTE]

> ====
> The `editor` {ref}/built-in-roles.html[built-in role] will grant write access
> to _all_ {kib} apps. If you want to limit access to the SLOs only, you will have
> to manually create and assign the mentioned roles.
> ====

Reviewer: ```suggestion
====
The `editor` {ref}/built-in-roles.html[built-in role] grants write access
to _all_ {kib} apps. If you want to limit access to the SLOs only, you have
to manually create and assign the mentioned roles.
====
```
Limiting the use of future tense and preferring the present tense where possible makes the reading lighter.

[discrete]
[[slo-all-access]]
== Create an SLO Editor role

Set the following privileges for the SLO Editor role:

. Under *Index privileges* in the *Elasticsearch* section, add `.slo-observability-*` to the *Indices* field and `read`, `view_index_metadata`, `write`, and `manage` to the *Privileges* field.
. Click *Add index privilege*.

> . Add indices you plan to create SLOs for to the *Indices* field, and add `read` and `view_index_metadata` to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.

Reviewer: ```suggestion
. In the *Indices* field, add the indices for which you plan to create SLOs. Then, add `read` and `view_index_metadata` to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.
```
